classes/music_album.rb

```diff
@@ -0,0 +1,29 @@
+require './item'
+
+class Album < Item
+  attr_reader :on_spotify
+
+  def initialize(publish_date, on_spotify)
+    super(publish_date, id)
```

Comment:
The parent class (Item) takes only one parameter (publish_date) the id will be generated automatically